recipes/default.rb

```diff
@@ -710,6 +711,30 @@
      username username
      admin_port admin_port
      secure false
+     not_if "#{asadmin} --user #{username} --passwordfile #{admin_pwd} list-jndi-resources | grep 'ldap/LdapResource'"
+  end
+end
+
+if node['kerberos']['enabled'].to_s == "true" && !node['kerberos']['krb_conf_path'].to_s.empty? 
+  krb_conf_path = node['kerberos']['krb_conf_path']
+  remote_file "#{theDomain}/config/krb5.conf" do
+    source "file:///#{krb_conf_path}"
+    owner node['glassfish']['user']
+    group node['glassfish']['group']
+    mode "0600"
+    action :create_if_missing
```

Comment:
I'd put these two as `action :create` so that if the configuration changes and/or the keytab regenerated the chef cookbook will automatically re-configure glassfish.